lib/linguist/languages.yml

```diff
@@ -153,6 +153,7 @@ Ant Build System:
   - ant.xml
   - build.xml
   ace_mode: xml
+  color: "#C64A9A"
```

Comment:
There's no point adding this as `type: data` languages don't contribute to repository stats (so the colour won't be visible in the language bar)